src/server/services/tests/BruincastServices.test.js

```diff
@@ -30,3 +30,132 @@ test('Test URL Generation', async done => {
+      timestamp: 1596157099334,
+    },
+    {
+      classShortname: '201C-EEBIOL162-1',
+      classID: '128672200',
+      term: '201',
+      date: '06/04/2020',
+      week: '10',
+      video: 'eeb162-1-20200604-18600.mp4',
+      audio: '',
+      title: 'Past Lectures',
+      comments:
+        '<p>Video recording is from Spring 2017 (date of lecture: 6/8/2017) ... No audio recording from Spring 2018 (6/7/2018)</p>\n',
+      timestamp: 1596157101968,
+    },
+    {
+      classShortname: '201C-EEBIOL162-1',
+      classID: '128672200',
+      term: '201',
+      date: '06/02/2020',
+      week: '10',
+      video: 'eeb162-1-20200602-18599.mp4',
+      audio: '',
+      title: 'Past Lectures',
+      comments:
+        '<p>Video recording is from Spring 2017 (date of lecture: 6/6/2017) ... No audio recording from Spring 2018 (6/5/2018)</p>\n',
+      timestamp: 1596157102195,
+    },
+  ];
+  const formattedArray = BruincastServices.formatCastListings(testDataArray);
+  const expectedArray = [
+    {
+      courseShortname: '201-COMSCI32-1',
+      courseListings: [
+        {
+          term: '20S',
+          classID: '187096200',
+          date: '05/11/2020',
+          title: 'Content is from CS 32 (Winter 2012)',
+          comments: '<p>Date of lecture: 3/14/2012</p>\n',
+          type: 'Video',
+          filename: 'cs32-1-20200511-18380.mp4',
+        },
+        {
+          term: '20S',
+          classID: '187096200',
+          date: '05/06/2020',
+          title: 'Content is from CS 32 (Winter 2012)',
+          comments: '<p>Date of lecture: 3/7/2012</p>\n',
+          type: 'Video',
+          filename: 'cs32-1-20200506-18379.mp4',
+        },
+        {
+          term: '20S',
+          classID: '187096200',
+          date: '05/04/2020',
+          title: 'Content is from CS 32 (Winter 2012)',
+          comments: '<p>Date of lecture: 3/5/2012</p>\n',
+          type: 'Video',
+          filename: 'cs32-1-20200504-18378.mp4',
+        },
+      ],
+    },
+    {
+      courseShortname: '201C-EEBIOL162-1',
+      courseListings: [
+        {
+          term: '201',
+          classID: '128672200',
+          date: '06/04/2020',
+          title: 'Past Lectures',
+          comments:
+            '<p>Video recording is from Spring 2017 (date of lecture: 6/8/2017) ... No audio&nbsp;recording&nbsp;from Spring 2018 (6/7/2018)</p>\n',
+          type: 'Video',
+          filename: 'eeb162-1-20200604-18600.mp4',
+        },
+        {
+          term: '201',
+          classID: '128672200',
+          date: '06/02/2020',
+          title: 'Past Lectures',
+          comments:
+            '<p>Video recording is from Spring 2017 (date of lecture: 6/6/2017) ... No audio&nbsp;recording&nbsp;from Spring 2018 (6/5/2018)</p>\n',
+          type: 'Video',
+          filename: 'eeb162-1-20200602-18599.mp4',
+        },
+      ],
+    },
+  ];
+  expect(formattedArray).toEqual(expectedArray);
```

Comment:
@chern  This appears to just test that type is being added... to make the test more interesting can you add an invalid comment comment, something that would get filtered by dompurify.sanitize to make sure that works as well?